### PR TITLE
    Use zconfig.Format for Storage Format in various structures instead of     string

### PR DESCRIPTION
**Pull request by kalyan-nidumolu**

### CODE REVIEW

File: pkg/pillar/cmd/domainmgr/domainmgr.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ import (
 
 	"github.com/eriknordmark/netlink"
 	"github.com/google/go-cmp/cmp"
+	zconfig "github.com/lf-edge/eve/api/go/config"
```
Comment (deitch) [inline pkg/pillar/cmd/domainmgr/domainmgr.go]: Why rename this to `zconfig` and not just leave it as the default `config`? It doesn't appear to conflict with another import.

Comment (kalyan-nidumolu) [reply pkg/pillar/cmd/domainmgr/domainmgr.go]: Was used similarly in other places.. hence did the same. IIUC - config is too generic a word.. zconfig clearly indicatesit is from the API..

Comment (deitch) [reply pkg/pillar/cmd/domainmgr/domainmgr.go]: OK. My preference is to stick to config, but I won't push too hard on it.

Comment (eriknordmark) [reply pkg/pillar/cmd/domainmgr/domainmgr.go]: Issue is that we have local variables in many files called config (and status). That seems to conflict with the package name.

File: pkg/pillar/cmd/domainmgr/domainmgr.go [hunk 2]
```diff
@@ -2598,7 +2600,7 @@ func createCloudInitISO(config types.DomainConfig) (*types.DiskStatus, error) {
 
 	ds := new(types.DiskStatus)
 	ds.ActiveFileLocation = fileName
-	ds.Format = "raw"
+	ds.Format = zconfig.Format_RAW
```
Comment (eriknordmark) [inline pkg/pillar/cmd/domainmgr/domainmgr.go]: Do you need a ToLower here as well?

File: pkg/pillar/cmd/zedmanager/zedmanager.go [hunk 3]
```diff
@@ -527,8 +527,6 @@ func handleCreate(ctxArg interface{}, key string,
 		ss := &status.StorageStatusList[i]
 		ss.UpdateFromStorageConfig(sc)
 		if ss.IsContainer {
-			// FIXME - We really need a top level flag to tell the app is
```
Comment (eriknordmark) [inline pkg/pillar/cmd/zedmanager/zedmanager.go]: Did it get less hacky somehow??

File: pkg/pillar/cmd/domainmgr/domainmgr.go (hunk 1 above)
Comment (deitch) [reply pkg/pillar/cmd/domainmgr/domainmgr.go]: Got it. I only looked at package names. Go is quite good about figuring out vars vs packages, but sometimes it cannot tell. Good to leave it then.

File: pkg/pillar/cmd/domainmgr/domainmgr.go (hunk 2 above)
Comment (kalyan-nidumolu) [reply pkg/pillar/cmd/domainmgr/domainmgr.go]: No.. Here it is no longer a string.. But the Const ( int )

File: pkg/pillar/cmd/zedmanager/zedmanager.go (hunk 3 above)
Comment (kalyan-nidumolu) [reply pkg/pillar/cmd/zedmanager/zedmanager.go]: I am no longer sure it is needed.. I guess I am more comfortable with the code now :-) My Earlier commits cleaned up a bunch of places where the Global flags were used. 

Hmm.. Thinking more, domain Mgr still needs to know the App is a container.. And there are other features coming - Like Environment Variable support - that are at the App Instance level ( Not just at storage level ) - So may be it is too early to remove this comment.

Let me remove this change.. As part of the Environment variable support, we will add that global flag..

File: pkg/pillar/cmd/domainmgr/domainmgr.go (hunk 2 above)
Comment (eriknordmark) [reply pkg/pillar/cmd/domainmgr/domainmgr.go]: OK

**Pull request merged**